controller_manager/src/controller_manager.cpp

```diff
@@ -3347,6 +3365,7 @@ void ControllerManager::controller_manager_diagnostic_callback(
   const std::string periodicity_stat_name = "periodicity";
   const auto cm_stats = periodicity_stats_.GetStatistics();
   stat.add("update_rate", std::to_string(get_update_rate()));
+  stat.add(periodicity_stat_name + ".sample_count", std::to_string(cm_stats.sample_count));
```

Comment:
I don't know if this is an intuitive diagnostic for the user, moreover it tend to be a constant rather than a varying value